openlibrary/plugins/upstream/utils.py

```diff
@@ -1222,13 +1222,16 @@ def _get_author_config():
 
     The results are cached on the first invocation.
     Any changes to /config/author page require restarting the app.
```

Comment:
The `/config/author` page isn’t getting loaded anymore, so restarting won’t do much… :)